paasta_tools/kubernetes_tools.py

```diff
@@ -1791,8 +1803,18 @@ def get_pod_template_spec(
             "paasta.yelp.com/service": self.get_service(),
             "paasta.yelp.com/instance": self.get_instance(),
             "paasta.yelp.com/git_sha": git_sha,
+            "paasta.yelp.com/autoscaled": str(self.is_autoscaling_enabled()).lower(),
+            "paasta.yelp.com/cluster": self.get_cluster(),
+            "paasta.yelp.com/pool": self.get_pool(),
+            # ideally, we'd also have the docker image here for ease-of-use
+            # in Prometheus relabeling, but that information is over the
+            # character limit for k8s labels (63 chars)
```

Comment:
why not just shorten it? I think docker does 12 by default?

Reply:
hmm, i'm a little wary of adding in a truncated form of this since that's not actually usable (e.g., if you try to docker pull with anything but the full tag docker will say there's nothing to pull)